### DISCUSSION TITLE
[@types/chrome] does not include all values for chrome.declarativeNetRequest.ResourceType enum

### DISCUSSION THREAD

**Discussion by hxnir** [DefinitelyTyped/DefinitelyTyped#71497] [Issues with a @types package]
Hey all,
First of all a huge thanks for this awesome types library!
I am configuring a chrome.declarativeNetRequest rule in my extension and I noticed that we are missing some potential values in the `ResourceType` enum.
Specifically the following values are missed
* `webtransport`
* `webbundle`

Link to the documentation specifying that these types exist - https://developer.chrome.com/docs/extensions/reference/api/declarativeNetRequest#enum_3.

will it be possible to add this values? 
I can open the PR myself if needed 😄

[a comment by an automated account is not shown]

**Comment by erwanjugand** [comment]
Relates to https://github.com/DefinitelyTyped/DefinitelyTyped/pull/71516
